smrf/data/loadTopo.py

```diff
@@ -57,72 +57,63 @@ def __init__(self, topoConfig, calcInput=True, tempDir=None):
         self._logger = logging.getLogger(__name__)
         self._logger.info('Reading [TOPO] and making stoporad input')
 
-        # read images
-        img_type = self.topoConfig['type']
-        if img_type == 'ipw':
-            #self.readImages()
-            raise IOError('IPW topos are deprecated as of SMRF 0.8.0')
```

Comment:
Will do. You just want a code comment or an actual exception raised?

Reply:
Since it's not an option in the config file, there is no way for someone to get to it without a lot of effort. I think removing it all together and making a note in the change log?

Reply:
Gotcha. Sounds good.